packages/p2p/src/hooks/__tests__/use-p2p-advertiser-adverts.spec.ts

```diff
@@ -0,0 +1,151 @@
+import { renderHook } from '@testing-library/react-hooks';
+import useP2PAdvertiserAdverts from '../use-p2p-advertiser-adverts';
+import { useStores } from 'Stores/index';
+
+jest.mock('@deriv/hooks', () => ({
+    ...jest.requireActual('@deriv/hooks'),
+    useP2PAdvertList: jest.fn(() => ({
+        data: [
+            {
+                account_currency: 'USD',
+                advertiser_details: {
+                    name: 'client Test90000253',
+                },
+                counterparty_type: 'buy',
+                country: 'id',
+                description: 'Created by script. Please call me 02203400',
+            },
+        ],
+    })),
+}));
+jest.mock('Stores', () => ({
+    ...jest.requireActual('Stores'),
+    useStores: jest.fn().mockReturnValue({}),
+}));
+
+const mockUseStores = useStores as jest.MockedFunction<typeof useStores>;
+
+describe('useP2PAdvertiserAdverts', () => {
+    it('should return the advertiser adverts object from response when is_advertiser_info_subscribed is true and counterparty_advertiser_id is defined', () => {
+        mockUseStores.mockReturnValue({
```

Comment:
@jim-deriv nit-pick :) The object you pass is the same in every test case, only some properties are changing, sometimes only 1 property. The object with values of `false`, `'test id 1234'`,  `'USD'`, `'buy'` is used twice. You might wanna create an object just once, and reuse it in the test cases.

Also, since `mockReturnValue` sets the return value for all further tests unless they get overriden, i.e. while you override all the properties in each test, it's gonna work fine. However, in case of adding more tests in future, new tests might be affected because of `mockReturnValue`, so you might wanna consider using `mockReturnValueOnce` which affects only 1 test case :) But it's totally up to you since all tests are currently passing.